Makefile

```
@@ -285,9 +285,13 @@ local-deploy: ## Deploy Kuadrant Operator in the cluster pointed by KUBECONFIG
 	@echo
 	@echo "Now you can export the kuadrant gateway by doing:"
 	@echo "kubectl port-forward -n istio-system service/istio-ingressgateway-istio 9080:80 &"
-	@echo "after that, you can curl -H \"Host: myhost.com\" localhost:9080"
-	@echo "-- Linux only -- Ingress gateway is exported using nodePort service in port 9080"
-	@echo "curl -H \"Host: myhost.com\" localhost:9080"
+	@echo "export GATEWAY_URL=localhost:9080"
+	@echo "after that, you can curl -H \"Host: myhost.com\" $$GATEWAY_URL"
+	@echo "-- Linux only -- Ingress gateway is exported using loadbalancer service in port 80"
+	@echo "export INGRESS_HOST=$$(kubectl get gtw istio-ingressgateway -n istio-system -o jsonpath='{.status.addresses[0].value}')"
+	@echo "export INGRESS_PORT=$$(kubectl get gtw istio-ingressgateway -n istio-system -o jsonpath='{.spec.listeners[?(@.name==\"http\")].port\}')"
```

Running this gives:

```
Now you can export the kuadrant gateway by doing:
kubectl port-forward -n istio-system service/istio-ingressgateway-istio 9080:80 &
export GATEWAY_URL=localhost:9080
after that, you can curl -H "Host: myhost.com" 172.18.200.0:80
-- Linux only -- Ingress gateway is exported using loadbalancer service in port 80
export INGRESS_HOST=172.18.200.0
error: error parsing jsonpath {.spec.listeners[?(@.name==\"http\")].port\}, unterminated filter
export INGRESS_PORT=
export GATEWAY_URL=172.18.200.0:80
curl -H "Host: myhost.com" 172.18.200.0:80
```

```suggestion
	@echo "export INGRESS_PORT=$$(kubectl get gtw istio-ingressgateway -n istio-system -o jsonpath='{.spec.listeners[?(@.name=="http")].port}')"
```

Seems `$GATEWAY_URL` is evaluated at the end of the command though since its always echoing to `172.18.200.0:80` is the above response 🤔

---

Aha yep, the `$` needs escaping twice, once for make and once for bash `echo`, will fix.